Creating use cases for "other components" (#1871)
saucelabs/sauce-docs: docs/api-testing/project-access.md

   <br/><img src={useBaseUrl('img/api-testing/notifications.png')} alt="Dashboard Pic 1.5"/>

:::tip

> Use the Fact component in your API tests to control the behavior of email notifications. This can be useful if you're testing in multiple environments. For more information, see [Fact](/api-testing/composer/other-components/#fact) and [Use Cases](/api-testing/use-cases/fact/).

Author: ```suggestion
Use the Fact component in your API tests to control the behavior of email notifications. It can be beneficial when you're testing in multiple environments. For more information, see [Fact](/api-testing/composer/other-components/#fact) and [Use Cases](/api-testing/use-cases/fact/).
```
